dockerfiles/theia/docker/alpine/runtime-install-dependencies.dockerfile

```diff
@@ -7,5 +7,5 @@
 # Install less for handling git diff properly
 # Install sshpass for handling passwords for SSH keys
 # Install lsblk as Theia requires it
-# Install libsecret as Theia requires it
-RUN apk add --update --no-cache sudo git bzip2 which bash curl openssh openssh-keygen less sshpass lsblk libsecret
+# Install libsecret-dev as Theia requires it
+RUN apk add --update --no-cache sudo git bzip2 which bash curl openssh openssh-keygen less sshpass lsblk libsecret-dev
```

Comment:
here we should keep libsecret, it's for the runtime

but in dev mode we should add libsecret-dev